src/Articulate/App_Plugins/Articulate/Themes/Phantom/Views/Master.cshtml

```diff
@@ -20,25 +20,24 @@
     <meta name="MobileOptimized" content="320" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    @{ Html
-            .RequiresJs("https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js")
-            .RequiresThemedCssFolder(Model);
+    @{ Smidge.RequiresJs("https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js")
```

Comment:
IIRC the preferred way to do this is to use bundles and render them that way. For example, this is in the vapor theme https://github.com/Shazwazza/Articulate/blob/release/5.0.0/src/Articulate/App_Plugins/Articulate/Themes/VAPOR/Views/Master.cshtml#L25

Reply:
That's great, didn't know that. 🙈